src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -36,10 +54,38 @@ export class PointerTool extends BaseTool {
       sceneController.hoveredGlyph = this.sceneModel.glyphAtPoint(point);
     }
 
-    this.setCursor();
+    if (!this.editor.visualizationLayersSettings.model["fontra.resize.selection"]) {
+      this.setCursor();
+      return;
+    }
+
+    const resizeHandle = this.getResizeHandle(event, sceneController.selection);
+    if (this.sceneController.sceneModel.hoverResizeHandle != resizeHandle) {
+      this.sceneController.sceneModel.hoverResizeHandle = resizeHandle;
+      this.canvasController.requestUpdate();
+    }
+    this.setCursorForResizeHandle(resizeHandle);
+  }
+
+  setCursorForResizeHandle(handleName) {
+    if (handleName === "bottom-left" || handleName === "top-right") {
+      this.setCursor("nesw-resize");
+    } else if (handleName === "bottom-right" || handleName === "top-left") {
+      this.setCursor("nwse-resize");
+    } else if (handleName === "bottom-center" || handleName === "top-center") {
+      this.setCursor("ns-resize");
+    } else if (handleName === "middle-left" || handleName === "middle-right") {
+      this.setCursor("ew-resize");
+    } else {
+      this.setCursor();
+    }
   }
 
-  setCursor() {
+  setCursor(cursor = undefined) {
+    if (cursor) {
+      this.canvasController.canvas.style.cursor = cursor;
+      return;
```

Comment:
Please remove the return, and make the "if" two lines below an "else if".

Reply:
I am happy to change it, but have you heard of the 'return early' code concept? This was my intention of putting it at the top:
https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
Of course I have, and I use it plenty.

In this case it is pretty much a switch between three cases. Two cases already existed, you added a third while not following the style of the code that was already there. Hence my feedback.

Reply:
done.